src/Enable.Extensions.Queuing.Abstractions/BaseQueueClient.cs

```diff
@@ -101,5 +144,14 @@ private IQueueMessage SerializeQueueMessage<T>(T content)
 
             return new QueueMessage(payload);
         }
+
+        private IQueueMessage SerializeQueueMessage<T>(T content, string sessionId)
```

Comment:
If dealing with the comment on EnqueueAsync, then this overload of this method that does not take a sessionId is no longer required. We should avoid the duplicated code in these overloads.